bridge.js

```diff
@@ -994,6 +1000,24 @@ Receiver.prototype = {
 Emitter(Receiver.prototype);
 
 /**
+ * Error object can't be sent via
+ * .postMessage() so we have to
+ * serialize them into an error-like
+ * Object that can be sent.
+ *
+ * @param  {*} err
+ * @return {Object|*}
+ */
+function serializeError(err) {
+  switch (err && err.constructor.name) {
+    case 'Error': return { message: err.message };
+    case 'DOMError': return { message: err.message, name: err.name };
+    case 'DOMException': return { message: err.toString() };
+    default: return err;
```

Comment:
@azasypkin I noticed that if an exception throws inside a Promise chain then the argument that gets passed to `.catch(err => ...` is a `DOMException` which is a little like a `String` but not cloneable.

I'm wondering if we should normalize all errors thrown in Services into a single string in the form of `DOMException`. I still don't know enough about the `DOMError`s thrown by your messaging service to know if this is a good idea.